qiskit-superstaq/qiskit_superstaq/superstaq_job.py

```diff
@@ -101,6 +101,17 @@ def result(self, timeout: Optional[float] = None, wait: float = 5) -> qiskit.res
             }
         )
 
+    def get_counts_on_qubits(
```

Comment:
My first thought:

The way to get counts in `qss` is through `job.result()`, so let's make this part of that method (right above).  So something like
```python
    def result(self, timeout: Optional[float] = None, wait: float = 5, qubits: Sequence[int]) -> qiskit.result.Result:
```

My second thought:

Can't a `SuperstaqJob` object hold multiple circuits?  What do you do if those circuits have different qubit numbers?  Maybe we actually need to pass a `Union[Sequence[int], Sequence[Sequence[int]]]` to cover the possibility of getting different qubits from different circuits...